packages/web-components/src/menu/menu.ts

```diff
@@ -15,4 +15,25 @@ export class Menu extends FASTMenu {
    */
   @attr({ mode: 'boolean' })
   public icons: boolean = false;
+  protected iconsChanged(): void {
+    if (!this.$fastController.isConnected) {
+      return;
+    }
+
+    this.menuItems?.filter(this.isMenuItemElement).forEach((item: HTMLElement, index: number) => {
```

Comment:
`menuItems` comes from the `FASTMenu` class?

Reply:
Ignore your squiggles...editors are only so smart: https://github.com/microsoft/fast/blob/d0ef0d6b1540ed90730187b45bdaf47aa9dcb183/packages/web-components/fast-foundation/src/menu/menu.ts#L34